src/NewTwitchApi/Resources/AdsApi.php

```diff
@@ -15,12 +15,12 @@ class AdsApi extends AbstractResource
      */
     public function startCommercial(string $bearer, string $broadcasterId, int $length): ResponseInterface
     {
-        $queryParamsMap = [];
+        $queryParamsMap = $bodyParamsMap = [];
```

Comment:
`$queryParamMap` is never changed. I'd rather you only define `$bodyParamsMap` here, then in the `postApi()` call at the end, just pass in `[]` instead of `$queryParamsMap`.